sys/docker/alpine-phpfpm/dev/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM php:7.4.11-fpm-alpine3.12
```

Comment:
I'd add a build arg PHP_VERSION with default value the one you need .. this way to test a different version, in local for example, you don't have to change the file and don't rick to commit it by mistake.